pro/src/components/Callout/__specs__/AddBankAccountCallout.spec.tsx

```diff
@@ -49,6 +49,12 @@ describe('AddBankAccountCallout', () => {
         venuesWithNonFreeOffersWithoutBankAccounts: [1],
         hasValidBankAccount: true,
       },
+      {
```

Comment:
Sur les tests c'est vraiment difficile à lire, la boucle it.each fait qu'on ne sait pas où on teste quoi, et l'intitulé n'est pas du tout exprimé en termes utilisateurs / métier.
Je propose d'enlever cette boucle et d'avoir pour chaque test un intitulé explicite
De même, pour `With FF enabled` il faudrait plutôt décrire à quoi correspond le FF